lib/charms/mysql/v0/mysql.py

```diff
@@ -979,6 +979,30 @@ def grant_privileges_to_user(
             logger.warning(f"Failed to grant privileges to user {username}@{hostname}", exc_info=e)
             raise MySQLGrantPrivilegesToUserError(e.message)
 
+    def update_user_password(
+        self, username: str, new_password: str, current_server_config_password: str
+    ) -> None:
+        """Updates user password in MySQL database."""
+        logger.debug(f"Updating password for {username}.")
+
+        connect_instance_address = self.instance_address
+        server_config_username = self.server_config_user
```

Comment:
these are redundant, just use `self.*` directly on the f-strings below

Reply:
ack